### PR TITLE
Homework 10 added

### PR DESCRIPTION
**Pull request by chicaragua**
I cannot get access to jump host from github action runner and used selfhosted runner

### CODE REVIEW

File: Ivan_Belov/10.Kubernetes.WS/config [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```
Comment (pluhin) [inline Ivan_Belov/10.Kubernetes.WS/config:1]: Please remove config, even commit with this config

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by chicaragua** [comment]
Done

File: Ivan_Belov/10.Kubernetes.WS/config (hunk 1 above)
Comment (chicaragua) [reply Ivan_Belov/10.Kubernetes.WS/config]: Done

**Comment by pluhin** [comment]
@chicaragua 
Please resolve secret issue

**Pull request closed without merging**